static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -28,12 +28,17 @@ const addScripts = (tags: Array<ThirdPartyTag>): void => {
         if (tag.beforeLoad) {
             tag.beforeLoad();
         }
-        if (tag.useImage === true) {
+        if (tag.useImage === true && typeof tag.url !== "undefined") {
             new Image().src = tag.url;
+        }
+        if (tag.insertSnippet) {
+            tag.insertSnippet();
         } else {
```

Comment:
This breaks the chain to `else` and inserts images as scripts! We need to fix this.

Reply:
Nop this doesn't break the chain. It still goes to the correct branch for images. The warning you see on the console is not related to this PR. 
It is because images are inserted with
`new Image().src = tag.url;`
instead of
`<img src={$tag.url}>`

Reply:
Depending on what facebook script expects it to be, it could be that we want it to convert it into a script tag as from the warning it implies it needs to be inserted as `script` and not an `image`

Reply:
Having misundertood the issue you are right we need an `else` before the ` if (tag.insertSnippet) {` so that it doesn't go to the script case

Reply:
https://github.com/guardian/frontend/pull/23381